test/documentation/CMakeLists.txt

```
@@ -0,0 +1,29 @@
+# Minimum cmake version
+cmake_minimum_required(VERSION 3.7)
+
+### Find b.i.o
+include (../../build_system/bio-config-version.cmake)
+set (BIO_VERSION "${PACKAGE_VERSION}")
+
+project (bio LANGUAGES NONE VERSION "${BIO_VERSION}")
+
+if (NOT EXISTS "${BIO_INCLUDE_DIR}/bio/version.hpp")
+    message (FATAL_ERROR "Could not find b.i.o. Not building documentation.")
+endif ()
+message (STATUS "Found: ${BIO_INCLUDE_DIR} (${BIO_VERSION})")
+
+set (BIO_DOXYGEN_INPUT_DIR "${CMAKE_SOURCE_DIR}")
+
+include (bio-doxygen.cmake)
```

```suggestion
include (../../submodules/seqan3/test/documentation/bio-doxygen.cmake)
```

instead of creating a symlink as suggest by my other comment